plugins/scaffolder/src/components/TaskPage/TaskPage.tsx

```diff
@@ -234,31 +240,38 @@ const hasLinks = ({ links = [] }: ScaffolderTaskOutput): boolean =>
  * @public
  */
 export type TaskPageProps = {
+  cancelTaskButtonText?: string;
+  cancellingTaskStatusText?: string;
```

Comment:
These might have been unnecessary at this point. If there's no direct use case for them immediately, it may be better to wait for proper i18n support to replace strings like these all over the place in my humble opinion. I see that there was already a text for loading, so I presume you were being diligent and following the pattern, so I'll ask @benjdlambert